whetstone/src/main/java/com/deliveryhero/whetstone/logging/InjectedComponent.kt

```diff
@@ -0,0 +1,13 @@
+package com.deliveryhero.whetstone.logging
+
+import android.app.Activity
+import android.app.Application
+import android.app.Service
+import android.view.View
+
+public sealed class InjectedComponent {
+    public class InjectedActivity(public val activity: Activity) : InjectedComponent()
```

Comment:
maybe also to reduce verbosity we can remove `Injected` prefix from all class names here? because they are already nested, and there's already `Injected` prefix in the outer class name. so maybe something like `InjectedComponent.Activity` would be less verbose?